.travis.yml

```diff
@@ -31,7 +31,7 @@ before_install:
 script:
   - sudo ln -sf /usr/bin/gcov-4.9 /usr/bin/gcov
   - bash -e tools/infrastructure/check_style.sh
-  - mkdir build && cd build && cmake ../ -DBUILD_TESTS=ON -DENABLE_GCOV=ON && make install
+  - mkdir build && cd build && cmake ../ -DBUILD_TESTS=ON -DENABLE_GCOV=ON && echo nproc is $("nproc") && make install -j$("nproc")
```

Comment:
Please extract  ```echo nproc is $("nproc")``` to separate line. And probably write it in more human readable. 
```
- ...
- echo " Number of processing units available :  $("nproc")"
- mkdir build && cd build  ...
```